lib/plugins/create/templates/aws-nodejs-docker/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Serverless directories
```

Comment:
Currently we name `.gitignore` as `gitignore` in template, as by default those files are not packaged by npm.

So either we can follow that here, or adjust our npm packaging rules, to ensure that `.gitignore` within templates is packaged. We can ensure that by adding  `!/lib/plugins/create/templates/**/.gitignore` to gitignore.

I would probably go with the latter, and also create an issue calling for refactor of `gitignore` -> `.gitignore` rename. It'll clear the confusion, and make templates handling a bit simpler

Reply:
Thanks - I was a bit confused to see `gitignore` instead of `.gitignore` in some packages - thanks for clearing that :bow: I'll adjust it :+1: